### ISSUE TITLE
Cooldown Number

### ISSUE THREAD

**Issue by Natlyz** [jordonwow/bigdebuffs#229]
**Hello 😊**

I have OmniCC that shows me the remaining time of buffs/debuffs/spells. And when I use it with big debuffs on the target/focus's portrait the number is HUGE, alsmost covering the whole icon making it hard to see the spell icon. I tried the size adjustment slider, but it doesn't seem to be working.

**Thanks!**

**Comment by domzae** [comment]
OmniCC has its own settings, you should be able to fix it there!

**Comment by Natlyz** [comment]
Yeah, OmniCC has a setting for font size, but then everything is same size.

e.g. 18 font size for the huge icon on the portraits, action bar and tiny debuffs on target/focus, when the number becomes bigger than the icon itself.

It's impossible to set cooldown text size for portrait icon separately 😢
